helm-charts/prometheus_operator_values.yaml

```diff
@@ -1841,7 +1857,7 @@ prometheus:
       limits:
         cpu: "3"
         memory: 15Gi
```

Comment:
@cpanato can we just increase this as well to be able to use the 64GB we will give?

Reply:
will add 60

Reply:
@stylianosrigas done PTAL